tests/basis_test.py

```diff
@@ -58,14 +27,16 @@ def test_cheb_call_degree_0_1():
         assert f0(i) == 1
         assert f1(i) == i
 
+
 def test_cheb_call_random_points():
```

Comment:
In addition to checking random points, might you also add a test where you check special points, like zeros or extrema, that have known values?